mern/server/package.json

```diff
@@ -5,21 +5,44 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest --coverage --detectOpenHandles",
```

Comment:
Does --coverage output the percentage of test coverage and whether there are OpenHandles? Where can we find the information output?